### DISCUSSION TITLE
Instructions for calling GPU to run input files

### DISCUSSION THREAD

**Discussion by biaogxb** [idaholab/moose#24014] [Q&A Getting Started] [answered]
I know the instruction to call the CPU to run the input.i files is'mpiexec -n 6 ./cat-opt -i input. i ', I want it to run faster. What is the instruction to call the GPU to run the input file?

**Comment by GiudGiud** [comment, marked as answer]
See https://github.com/idaholab/moose/discussions/21152

We dont have a full set of instruction anywhere else
